### PR TITLE
add shortcode to training page

### PR DESCRIPTION
**Pull request by Arhell**
this PR https://github.com/kubernetes/website/pull/24014

### CODE REVIEW

**Comment by Arhell** [comment]
review page 
https://deploy-preview-24951--kubernetes-io-master-staging.netlify.app/pl/training/
/assign @mfilocha

File: content/pl/training/_index.html [hunk 1]
```diff
@@ -112,7 +112,6 @@ <h2>Partnerzy szkoleniowi Kubernetes</h2>
     </center>
   </div>
   <div class="main-section landscape-section">
-    <iframe src="https://landscape.cncf.io/category=kubernetes-training-partner&format=logo-mode&grouping=category&embed=yes" frameborder="0" id="landscape" scrolling="no"></iframe>
-    <script src="https://landscape.cncf.io/iframeResizer.js"></script>
+    {{< cncf-landscape helpers=false category="kubernetes-training-partner" >}}
```
Comment (nvtkaszpir) [inline content/pl/training/_index.html:115]:
```suggestion
    {{< cncf-landscape helpers="false" category="kubernetes-training-partner" >}}
```

**Review by nvtkaszpir — COMMENTED** [review]
Shouldn't this be quoted?

File: content/pl/training/_index.html (hunk 1 above)
Comment (Arhell) [reply content/pl/training/_index.html:115]: No, I tested in the en version when I did the PR, there was an error
https://github.com/kubernetes/website/pull/24014

Comment (nvtkaszpir) [reply content/pl/training/_index.html:115]: huh ;D

**Pull request merged**